securedrop/source_templates/index.html

```diff
@@ -17,7 +20,7 @@
     <div class="content">
       <div class="grid">
         <div id="header">
-          <img src="/static/i/{{ header_image }}" class="logo grid-item" alt="SecureDrop">
+          <img src="/static/i/{{ header_image }}" class="logo grid-item" alt="SecureDrop" width="150px" height="135px">
```

Comment:
The width/height are wrong, and the image looks distorted as a result:

<img width="603" alt="screen shot 2015-11-08 at 1 09 26 pm" src="https://cloud.githubusercontent.com/assets/407302/11022702/fe1f2eaa-8619-11e5-9023-97494a34c3fc.png">

According to `source.css`, width should be 250px. Height should be whatever maintains the aspect ratio to that width.

Note that the actual `logo.png` has dimensions that are twice as large as this (500x450 px, documented [here](https://docs.securedrop.org/en/latest/install.html#configure-the-installation)). This is intentional, so the web interfaces look good on computers with HiDPI displays (e.g. Apple Retina).